components/profile-loader/profile-loader.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
```

Comment:
Logic here is pretty much the same as `project-loader.jsx` except it's a lot simpler.